scripts/run-e2e-tests.sh

```diff
@@ -0,0 +1,207 @@
+  helm repo add yunikorn https://apache.github.io/incubator-yunikorn-release && helm repo update
+  exit_on_error "add yunikorn helm repo failed"
+
+  # load latest yunikorn docker images to kind
+  kind load docker-image local/yunikorn:scheduler-latest --name ${k8s_cluster_name}
+  kind load docker-image local/yunikorn:admission-latest --name ${k8s_cluster_name}
+
+  kubectl create namespace yunikorn
+  exit_on_error "failed to create yunikorn namespace"
+  helm install yunikorn yunikorn/yunikorn --namespace yunikorn \
+    --set image.repository=local/yunikorn \
+    --set image.tag=scheduler-latest \
+    --set image.pullPolicy=Never
+  exit_on_error "failed to install yunikorn"
+  kubectl wait --for=condition=available --timeout=300s deployment/yunikorn-scheduler -n yunikorn
+  exit_on_error "failed to wait for yunikorn scheduler deployment being deployed"
+  kubectl wait --for=condition=ready --timeout=300s pod -l app=yunikorn -n yunikorn
+  exit_on_error "failed to wait for yunikorn scheduler pods being deployed"
+
+  # forward rest server port 9080, so the e2e test code can verify cluster state via rest calls
+  kubectl port-forward svc/yunikorn-service 9080:9080 -n yunikorn &
+  exit_on_error "failed to forward rest server port 9080"
+  kubectl describe deployment yunikorn-scheduler -n yunikorn
+  exit_on_error "failed to describe yunikorn scheduler deployment"
+}
+
+function delete_cluster() {
+  if [[ $# -ne 1 ]]; then
+    echo "expecting exact 1 parameters for function install_cluster()"
+    return 1
+  fi
+
+  k8s_cluster_name=$1
+
+  echo "step 1/2: deleting yunikorn scheduler helm charts"
+  helm uninstall yunikorn --namespace yunikorn
+  exit_on_error "failed to uninstall helm charts"
+  echo "step 2/2: deleting K8s cluster: ${k8s_cluster_name}"
+  kind delete cluster --name ${k8s_cluster_name}
+  exit_on_error "failed to delete the cluster"
+
+}
+
+function print_usage() {
+    cat <<EOF
+Usage: $(basename "$0") -a <action> -n <kind-cluster-name> -v <kind-node-image-version>
+  <action>                     the action that needs to be executed, must be either "test" or "cleanup".
+  <kind-cluster-name>          the name of the K8s cluster that created by kind.
+  <kind-node-image-version>    the kind node image used to provision the K8s cluster.
+
+Examples:
+  $(basename "$0") -n "yk8s" -v "kindest/node:v1.15.3"
+EOF
+}
+
+while [[ $# -gt 0 ]]; do
+key="$1"
+case ${key} in
+  -a|--action)
+    action="$2"
+    shift
+    shift
+    ;;
+  -n|--cluster-name)
+    cluster_name="$2"
+    shift
+    shift
+    ;;
+  -v|--cluster-version)
+    cluster_version="$2"
+    shift
+    shift
+    ;;
+  -h|--help)
+    print_usage
+    exit 0
+    ;;
+  *)
+    echo "unknown option: ${key}"
+    print_usage
+    exit 1
+    ;;
+esac
+done
+
+echo "action: ${action}"
+check_opt "${action}"
+echo "kind cluster name: ${cluster_name}"
+check_opt "${cluster_name}"
+echo "kind node image version ${cluster_version}"
```

Comment:
since "cluster_version" is not passed when this script is invoked, it will be always empty string!